### PR TITLE
Task: Switch to .css only

### LINKED ISSUE #187: Add .css only version of header and footer

### PR DESCRIPTION
**Pull request by paulathevalley**
Resolves #187 
Gonna be a major bump!

### CODE REVIEW

File: src/styles/components/header.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.formidableHeader .default * {
+  display: inline-block;
+  margin-left: 2em;
+}
+
+/** DARK THEME
+*/
+.formidableHeader.isDark {
+  background: var(--black); /* TODO: Fix assets url("../../assets/bg.jpg") bottom right; */
```
Comment (paulathevalley) [inline src/styles/components/header.css]: :thinking: When linking `formidable-landers` in `victory-docs`, the `url("../../assets.bg.jpg")` fails... I think this is similar to https://github.com/FormidableLabs/victory-docs/issues/215

```
ERROR in ./~/css-loader!./~/postcss-loader!../formidable-landers/lib/styles/styles.css
Module not found: Error: Cannot resolve 'file' or 'directory' ../../assets/bg.jpg in
/Users/paula/work/formidable/code/formidable-landers/lib/styles
 @ ./~/css-loader!./~/postcss-loader!../formidable-landers/lib/styles/styles.css 6:3940-3970
```

formidable-landers/lib/:
screen shot 2016-11-14 at 3 22 49 pm

**Comment by paulathevalley** [comment]
This version should be used in combination with https://github.com/FormidableLabs/builder-docs-archetype/pull/38

**Comment by exogen** [comment]
LGTM. Is there any reason all the webpack stuff is in here too instead of being in an archetype?

**Comment by paulathevalley** [comment]
@exogen Hmm, this repo doesn’t depend on the archetype, since the build step is just running `babel` and doesn’t generate any static files. Should it depend on the archetype? 

The webpack file here is just to run a demo page to see the `<Header>` and `<Footer>` components.

**Pull request merged**